docs/html/_sources/experimentsetup.rst.txt

```diff
@@ -0,0 +1,27 @@
+Experiment Setup
```

Comment:
what are these rst.txt files in _source?  Should they be under git control or are they created by the build or what are they?  @chiahaoliu do you know?

Reply:
I am actually not sure about this. ``rever`` seems to keep it(https://github.com/xpdAcq/xpdAcq/tree/gh-pages)

I removed them from the source and the website still works fine.